package.json

```diff
@@ -11,10 +11,14 @@
   "scripts": {
     "ci:build": "ts-node tasks/build",
     "test": "mocha",
+    "test:integration": "node index.js --build -i test/integration/index.js -o integration-tests --verbose -r test/integration -r node_modules/ && npm run test:integration:run",
+    "test:integration:run": "run-script-os",
```

Comment:
i don't know if there's a better way to do this? `./integration-tests` seems to fail on Windows; whilst `integration-tests` won't work on Linux